### DISCUSSION TITLE
Upgraded to SH beta, suddenly all emails go to spam (Gmail)

### DISCUSSION THREAD

**Discussion by georgegebbett** [simple-login/app#1234] [Self-hosting question] [answered]
Hello! 

I have been tearing my hair out over this one for a while now and have completely failed to make any progress on what is going wrong. 

I recently updated to the latest beta and ever since doing this, every single email that comes through SL is being sent to spam by Gmail. 

DKIM, DMARC and SPF are all passing according to Gmail, and this happens over each domain I have set up for aliases. There is nothing untoward in the logs. 

I have attached the three test emails sent to three different domains which all went to spam - v confused as to what is going on and if anyone can shed any light I would be v appreciative. 

[Test emails.zip](https://github.com/simple-login/app/files/9351228/Test.emails.zip)

**Comment by nguyenkims** [comment, marked as answer]
@georgegebbett The new SL version doesn't change the way email headers are generated so that's indeed strange. I wonder if your domain or your IP address has been blacklisted anywhere? If not and if the emails pass DKIM, DMARC, SPF, there should be a low chance that they end up in Spam.

**Reply by georgegebbett** [reply]
Oddly, the problem stopped as abruptly as it started. I did reach out to Google, but given the generic response I received from one of their first line agents, I doubt that had anything to do with it - one of life's little mysteries!

**Reply by nguyenkims** [reply]
How and why email providers block emails is odd and doesn't seem to be an exact science :).

**Comment by trikotret** [comment]
what Beta
